validphys2/src/validphys/pseudodata.py

```diff
@@ -129,6 +129,7 @@ def make_replica(
     sep_mult,
     genrep=True,
     max_tries=int(1e6),
+    resample_pseudodata=True,
```

Comment:
```suggestion
    resample_negative_pseudodata=True,
```

I think this way it is clearer what is happening.